ironfish/src/memPool/memPool.ts

```diff
@@ -173,30 +173,31 @@ export class MemPool {
     let addedTransactions = 0
 
     for (const transaction of block.transactions) {
-      if (this.exists(transaction.hash())) {
-        continue
-      }
-
       if (transaction.isMinersFee()) {
         continue
       }
 
-      this.addTransaction(transaction)
-      addedTransactions++
+      const added = this.addTransaction(transaction)
+      if (added) {
```

Comment:
anything we should do if `added` is null ?

Reply:
Will that message be useful to anyone? It's expected to print that a lot, so it'll print it many many times but there's no actionable thing you can do with that message I think because it's expected.

Reply:
That will print 300~ lines every block that's reorged.